.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+	"presets": [
+		[
+			"@babel/preset-env",
+			{
+				"targets": {
+					"node": "current"
+				}
+			}
+		]
+	],
+	"plugins": ["@babel/plugin-proposal-class-properties"]
```

Comment:
Pretty sure this is only needed for examples

Reply:
It is needed for class properties in general